core/src/main/scala/akka/persistence/cassandra/CassandraPluginConfig.scala

```diff
@@ -37,7 +39,8 @@ class CassandraPluginConfig(system: ActorSystem, config: Config) {
   val replicationStrategy: String = getReplicationStrategy(
     config.getString("replication-strategy"),
     config.getInt("replication-factor"),
-    config.getStringList("data-center-replication-factors").asScala)
+    Try(config.getStringList("data-center-replication-factors").asScala)
```

Comment:
c.getValue("data-center-replication-facotors").valueType() match {
      case ConfigValueType.STRING ⇒
      case ConfigValueType.STRING_LIST ⇒
    }

slightly nicer to avoid exceptions

Reply:
Nice idea! I will try to apply it.